### ISSUE TITLE
minPixelRatio

### ISSUE THREAD

**Issue by zmomozq** [openseadragon/openseadragon#2385]
If minPixelRatio is set to 1, the quality of image rendering will be affected, but if it is not set to 1 and the default value of 0.5 is kept, it will be slow in the first loading.  Is there a way to dynamically set minPixelRatio and render a fuzzy level when it is turned on?  But it is also possible to load higher levels when scaling

**Comment by iangilman** [comment]
We don't have an explicit function for setting it after startup, but you can do so like so: 

```
function setMinPixelRatio(value) {
    const count = viewer.world.getItemCount();
    for (let i = 0; i < count; i++) {
        const tiledImage = viewer.world.getItemAt(i);
        tiledImage.minPixelRatio = value;
    }
}
```

I suppose we could add such a function if you or anyone wants to make a PR for it.

**Comment by zmomozq** [comment]
> 我们没有在启动后设置它的显式函数，但您可以这样做：
> 
> ```
> function setMinPixelRatio(value) {
>     const count = viewer.world.getItemCount();
>     for (let i = 0; i < count; i++) {
>         const tiledImage = viewer.world.getItemAt(i);
>         tiledImage.minPixelRatio = value;
>     }
> }
> ```
> 
> 我想如果你或任何人想为其制作 PR，我们可以添加这样的功能。

Thank you for your reply.  I think I can try it when I have time

**Comment by zmomozq** [comment]
> > 我们没有在启动后设置它的显式函数，但您可以这样做：
> > ```
> > function setMinPixelRatio(value) {
> >     const count = viewer.world.getItemCount();
> >     for (let i = 0; i < count; i++) {
> >         const tiledImage = viewer.world.getItemAt(i);
> >         tiledImage.minPixelRatio = value;
> >     }
> > }
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 我想如果你或任何人想要制作公关，我们可以添加这样的功能。
> 
> 感谢你的回复。我想我有时间的时候可以尝试一下



> 我们没有在启动后设置它的显式函数，但您可以这样做：
> 
> ```
> function setMinPixelRatio(value) {
>     const count = viewer.world.getItemCount();
>     for (let i = 0; i < count; i++) {
>         const tiledImage = viewer.world.getItemAt(i);
>         tiledImage.minPixelRatio = value;
>     }
> }
> ```
> 
> 我想如果你或任何人想为其制作 PR，我们可以添加这样的功能。

Sorry, there's really no need to add such a feature. Setting immediateRender to false will load the blurry image first, followed by the slightly higher level image.

**Comment by iangilman** [comment]
Cool. Sounds like you got it sorted out, then?

**Comment by zmomozq** [comment]
> Cool. Sounds like you got it sorted out, then?
Yes, set the immediateRender property to false, when it is turned on, it will first load tiles with a lower level for rendering, and then load tiles with a slightly higher level
